test/parallel/test-stream2-unpipe-drain.js

```diff
@@ -30,12 +27,11 @@ function TestReader(id) {
 util.inherits(TestReader, stream.Readable);
 
 TestReader.prototype._read = function(size) {
-  this.reads += 1;
-  this.push(crypto.randomBytes(size));
+  this.push(Buffer.alloc(size));
```

Comment:
If this isn't using crypto to push data to the stream, the `common.hasCrypto` check is still needed?